### PR TITLE
[SW-1156] Split the TransactionList logic between parent components

### PR DESCRIPTION
**Pull request by peronczyk**

### CODE REVIEW

File: src/popup/pages/FungibleTokens/TokenTransactions.vue [hunk 1]
```diff
@@ -18,5 +28,72 @@ export default defineComponent({
...
     isMultisig: Boolean,
   },
+  setup(props) {
+    const store = useStore();
+
+    const { activeAccount } = useAccounts({ store });
+    const {
+      fetchTransactions,
+      getAccountAllTransactions,
+      // getAccountTransactionsState,
```
Comment (pstachel) [inline src/popup/pages/FungibleTokens/TokenTransactions.vue:38]: can be removed

File: src/protocols/aeternity/views/AccountDetailsTransactions.vue
Comment (pstachel) [inline src/protocols/aeternity/views/AccountDetailsTransactions.vue:1]: Isn't here a lot of duplicated code from AccountDetailsMultisigTransactions?

**Review by pstachel — APPROVED** [review]

File: src/protocols/aeternity/views/AccountDetailsTransactions.vue
Comment (peronczyk) [reply src/protocols/aeternity/views/AccountDetailsTransactions.vue:1]: Yes, totally. I would definitely clean this but we don't have much time. I wanted to make less damage.

Comment (pstachel) [reply src/protocols/aeternity/views/AccountDetailsTransactions.vue:1]: please leave a comment there suggesting future improvements

Comment (peronczyk) [reply src/protocols/aeternity/views/AccountDetailsTransactions.vue:1]: Done, added comments.

**Pull request merged**